File: web-deteccion/src/app/royal/modulos/framework-comun/core/dominio/filtro/FiltroComunTipocambiomast.ts
import { DominioPaginacion } from '../../../../../framework/modelo/generico/DominioPaginacion';


export class FiltroComunTipocambiomast{

    constructor() {
        this.paginacion = new DominioPaginacion();
      }
    
	fechacambio: Date;		
	estado: string;	
    
    paginacion: DominioPaginacion;


    monedacodigo : string;
    monedacambiocodigo : string;
 
    fechacambiostring : string;
    factor : number;
    factorcompra : number;
    factorventa : number;
    factorpromedio : number;
    factorcompraafp : number;
    factorventaafp : number;
    factorcomprasbs : number;
    factorventasbs : number;
    valorcuota : number;
  
    ultimafechamodif : Date;
    ultimousuario : string;
    tasatamex : number;
    tasatamn : number;
    tasaanualtamex : number;
    tasaanualtamn : number;
    tipoexportar:string

}

File: blinkeye-framewk-comun-dominio/src/main/java/net/royal/spring/core/dominio/filtro/FiltroComunTipocambiomast.java
package net.royal.spring.core.dominio.filtro;

import net.royal.spring.core.dominio.BeanTipocambiomast;

import java.math.BigDecimal;
import java.util.Date;
import javax.persistence.Column;
import javax.persistence.EmbeddedId;
import javax.persistence.Entity;
import javax.persistence.Table;
import javax.persistence.Transient;
import javax.validation.constraints.NotEmpty;
import javax.validation.constraints.NotNull;
import javax.validation.constraints.Pattern;
import javax.validation.constraints.Size;

import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import com.fasterxml.jackson.databind.annotation.JsonSerialize;

import net.royal.spring.framework.modelo.generico.DominioPaginacion;
import net.royal.spring.framework.modelo.generico.DominioTransaccion;
import net.royal.spring.framework.web.rest.UDeserializers;
import net.royal.spring.framework.web.rest.USerializers;

/**
 * Tipo de cambio diario
 * 
 * @tabla TIPOCAMBIOMAST
*/
public class FiltroComunTipocambiomast implements java.io.Serializable{

	private String estado;
	private java.util.Date fechacambio;
	private String monedacodigo;
	private String monedacambiocodigo;
	private String fechacambiostring;
	private String tipoexportar;
	private DominioPaginacion paginacion;
	
	public String getMonedacodigo() {
		return monedacodigo;
	}

	public void setMonedacodigo(String monedacodigo) {
		this.monedacodigo = monedacodigo;
	}

	public String getMonedacambiocodigo() {
		return monedacambiocodigo;
	}

	public void setMonedacambiocodigo(String monedacambiocodigo) {
		this.monedacambiocodigo = monedacambiocodigo;
	}

	public String getFechacambiostring() {
		return fechacambiostring;
	}

	public void setFechacambiostring(String fechacambiostring) {
		this.fechacambiostring = fechacambiostring;
	}

	public String getTipoexportar() {
		return tipoexportar;
	}

	public void setTipoexportar(String tipoexportar) {
		this.tipoexportar = tipoexportar;
	}

	public String getEstado() {
		return estado;
	}

	public void setEstado(String estado) {
		this.estado = estado;
	}

	public java.util.Date getFechacambio() {
		return fechacambio;
	}

	public void setFechacambio(java.util.Date fechacambio) {
		this.fechacambio = fechacambio;
	}



	public DominioPaginacion getPaginacion() {
		if (paginacion == null)
			paginacion = new DominioPaginacion();
		return paginacion;
	}

	public void setPaginacion(DominioPaginacion paginacion) {
		this.paginacion = paginacion;
	}	
}
